File: games/lia-1/bots/java/src/core/api/MatchState.java
package core.api;

public class MatchState {
    public int __uid;
    public float time;
    public int numberOfOpponentUnits;
    public int resources;
    public boolean canSaySomething;
    public UnitData[] units;

    public int getUid() {
        return __uid;
    }
}

File: games/planetization/bots/kotlin/src/core/api/MatchState.kt
package core.api

import java.util.ArrayList

class MatchState(var __uid: Int,
                 var time: Float,
                 var yourPlanets: ArrayList<PlanetData>,
                 var freePlanets: ArrayList<PlanetData>,
                 var opponentPlanets: ArrayList<PlanetData>,
                 var yourWorkers: ArrayList<UnitData>,
                 var opponentWorkers: ArrayList<UnitData>,
                 var yourWarriors: ArrayList<UnitData>,
                 var opponentWarriors: ArrayList<UnitData>
)
